map_app/map_app/MapViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  MapViewController.swift
+//  map_app
+//
+//  Created by Allison Collins on 2/11/20.
+//  Copyright © 2020 Allison Collins. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+class MapViewController: UIViewController {
+    @IBOutlet var mapView: MKMapView! {
+        didSet { mapView.delegate = self }
+    }
+
+    @IBOutlet var pointDescription: pointView!
+    @IBOutlet var favoritesButton: UIButton!
+    var myAnnotations = [Place]()
+    
+    @IBAction func newFave(_ sender: UIButton) {
+        let name = self.pointDescription.placeLabel.text!
+        DataManager.sharedInstance.saveFavorites(name: name)
+        sender.isSelected = true
+    }
+    @IBAction func switchView(_ sender: Any) {
```

Comment:
Use consistent new lines between new functions. Here you use none, but throughout the rest of your code, you use 1. 1 is ideal.